features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Comments/Comment.jsx

```diff
@@ -190,36 +191,37 @@ class Comment extends React.Component {
         const {
             apiId, allComments, commentsUpdate, intl,
         } = this.props;
-        const commentIdOfCommentToDelete = deleteComment.commentId;
+        const commentIdOfCommentToDelete = deleteComment.id;
         const parentCommentIdOfCommentToDelete = deleteComment.parentCommentId;
         this.handleClose();
 
         apiClient.deleteComment(apiId, commentIdOfCommentToDelete)
             .then((result) => {
-                if (parentCommentIdOfCommentToDelete === undefined) {
+                // if (parentCommentIdOfCommentToDelete === undefined) {
                     const remainingComments = allComments.filter(this.filterRemainingComments);
                     commentsUpdate(remainingComments);
-                } else {
-                    const index = allComments.findIndex(this.filterCommentToDelete);
-                    const remainingReplies = allComments[index].replies.filter(this.filterRemainingComments);
-                    allComments[index].replies = remainingReplies;
-                    commentsUpdate(allComments);
-                }
+                    Alert.message("Comment" + commentIdOfCommentToDelete + "has been successfully deleted");
+                // } else {
+                //     const index = allComments.findIndex(this.filterCommentToDelete);
+                //     const remainingReplies = allComments[index].replies.filter(this.filterRemainingComments);
+                //     allComments[index].replies = remainingReplies;
+                //     commentsUpdate(allComments);
+                // }
             })
             .catch((error) => {
                 console.error(error);
-                if (error.response) {
-                    Alert.error(error.response.body.message);
-                } else {
-                    Alert.error(
-                        intl.formatMessage({
-                            defaultMessage: 'Something went wrong while deleting comment',
-                            id: 'Apis.Details.Comments.Comment.something.went.wrong',
-                        })
-                            + ' - '
-                            + commentIdOfCommentToDelete,
-                    );
-                }
+                // if (error.response) {
```

Comment:
Shouldn't we keep these Alerts to identify failures?

Reply:
fixed